### ISSUE TITLE
prettier/prettier ESLint errors for new lines

### ISSUE THREAD

**Issue by seangwright** [prettier/eslint-plugin-prettier#298]
**What version of `eslint` are you using?**
`"eslint": "7.1.0",`
(also tried with `"eslint": "6.8.0",`)

**What version of `prettier` are you using?**
`"prettier": "2.0.5",`

**What version of `eslint-plugin-prettier` are you using?**
`"eslint-plugin-prettier": "3.1.3",`
`"eslint-config-prettier": "6.11.0",`

**Please paste any applicable config files that you're using (e.g. `.prettierrc` or `.eslintrc` files)**

`./.prettierrc.json`
```json
{
  "bracketSpacing": true,
  "endOfLine": "auto",
  "htmlWhitespaceSensitivity": "css",
  "printWidth": 80,
  "semi": true,
  "singleQuote": true,
  "tabWidth": 2,
  "useTabs": false,
  "trailingComma": "all"
}
```

`./.estlintrc.json`
```json
{
  "root": true,
  "env": {
    "node": true
  },
  "extends": [
    "plugin:prettier/recommended"
  ],
  "globals": {},
  "rules": {},
  "parserOptions": {
    "parser": "babel-eslint",
    "ecmaVersion": 10,
    "sourceType": "module"
  },
  "settings": {
    "import/extensions": [".js", ".vue"]
  }
}
```

**What source code are you linting?**
```js
import {
  dom,
  library,
} from '@fortawesome/fontawesome-svg-core';

// more lines below that lint correctly
```

**What did you expect to happen?**
ESLint formats the document based on my Prettier configuration

**What actually happened?**
![Annotation 2020-05-26 190022](https://user-images.githubusercontent.com/1382768/82958323-2cd65e80-9f83-11ea-8d4e-12a44bba5578.jpg)

ESLint reports errors 

```json
{
	"resource": "/path/to/file/icon-library.js",
	"owner": "eslint",
	"code": {
		"value": "prettier/prettier",
		"target": {
			"$mid": 1,
			"external": "https://github.com/prettier/eslint-plugin-prettier#options",
			"path": "/prettier/eslint-plugin-prettier",
			"scheme": "https",
			"authority": "github.com",
			"fragment": "options"
		}
	},
	"severity": 8,
	"message": "Replace `␍⏎··dom,␍⏎··library,␍⏎` with `·dom,·library·`",
	"source": "eslint",
	"startLineNumber": 1,
	"startColumn": 9,
	"endLineNumber": 4,
	"endColumn": 1
}
```

**Additional versions**

VS Code: 1.45.1
ESLint Extension: 2.1.5 https://github.com/Microsoft/vscode-eslint
Prettier Extension: 4.7.0 https://github.com/prettier/prettier-vscode

**Additional notes**

If I change my Prettier config to `"trailingComma": "es5"` then I get errors after the last token of each import statement set (eg: `import { dom, library } from ...` errors after `library` saying `Insert ',' eslint(prettier/prettier)`).

It's almost like my Prettier configuration isn't being respected 🤔.

**Comment by j-f1** [comment]
That’s the intended behavior. If you run `eslint --fix` it’ll format your code to Prettier’s standards.

**Comment by seangwright** [comment]
Hm. When I save the file in VS Code and it formats but formats into an invalid state, I'm in a bit of a bind.

Am I missing something that I should be doing to have `formatOnSave: true` in VS Code result in a document that passes linting?

**Comment by lydell** [comment]
@seangwright Step one is to make sure that everything works when using the CLI. Then move on to debugging VSCode.

**Comment by pastorcash** [comment]
I had the same issue just start with me when I created a new .vue file.
After some research, I believe that it may be to the difference between the CR, LF, and CRLF issue
I added the following rule to my eslintrc,js configuration file:
    'linebreak-style': [2, "windows"]
I am using a windows operating system.
For  a more detailed description of this rule see the following
https://eslint.org/docs/2.0.0/rules/linebreak-style
Hope this helps, at least temporarily.
